.github/workflows/nix.yml

```
@@ -1,16 +1,32 @@
-name: Continuous Integration (Nix)
+name: UKCI
 
 permissions:
   contents: read
 
+.trigger_paths: &trigger_paths
```

actionlint (1.7.11)

[error] unexpected key ".trigger_paths" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)